cmd/podman/libpodruntime/runtime.go

```diff
@@ -8,8 +8,17 @@ import (
 	"github.com/pkg/errors"
 )
 
+// GerRuntimeRenumber gets a libpod runtime that will perform a lock renumber
```

Comment:
I'm surprised validate didn't choke on this.  s/GerRuntimeRenumber/GetRuntimeRenumber/

Reply:
Fixed